EBA/Taschenrechner/Inforechner/fm_BitByte.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Data;
+using System.Drawing;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows.Forms;
+
+namespace JustForYou_Informatik
+{
+    public partial class fm_BitByte : Form
+    {
+        public fm_BitByte()
+        {
+            InitializeComponent();
+
+        }
+
+        private void btn_Weiter_Click(object sender, EventArgs e)
+        {
+            var eingabe = 800; //Testvariable,später durch Wert aus Eingabemosul ersetzen
```

Comment:
todo: this isn't ready now. Please put a `//TODO: describtion` near this.

Reply:
this whole class got rm again ... @JanMichels  learn 2 merge